x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/FieldExtractorTestCase.java

```diff
@@ -1435,7 +1436,14 @@ private String deyaml(String err) {
     }
 
     private static Map<String, Object> runEsql(String query) throws IOException {
-        return runEsqlSync(new RestEsqlTestCase.RequestObjectBuilder().query(query).version(EsqlTestUtils.latestEsqlVersionOrSnapshot()));
+        String versionString = null;
+        Set<EsqlVersion> versions = EsqlSpecTestCase.availableVersions();
+        if (versions.isEmpty() == false) {
+            // Use the latest released version or SNAPSHOT, if available.
+            versionString = versions.stream().max(Comparator.comparingInt(EsqlVersion::id)).toString();
+        }
```

Comment:
We were sending a version in the JSON body on bwc tests; the version parameter is only available on 8.13.3+, though.

Reply:
C.f. corresponding entry in `build.gradle`: https://github.com/elastic/elasticsearch/blob/f7349e205e159fa39d96114c13b12c57cf2e7d73/x-pack/plugin/esql/qa/server/mixed-cluster/build.gradle#L33-L36